imports of wasm-bindgen file reports : No loader is configured for ".wasm" files.

### Describe the bug

npm package using any *_bg.wasm does not import.
error looks like: 
```
 'ERROR: No loader is configured for ".wasm" files: ...@dqbd/tiktoken/tiktoken_bg.wasm.'
```

vite.config.js:
```
export default defineConfig({
    plugins: [sveltekit(),wasm(),topLevelAwait()]
}
```

### Reproduction URL

none

### Reproduction

npm run build

### Logs

```shell
> Using @sveltejs/adapter-cloudflare
✘ [ERROR] No loader is configured for ".wasm" files: node_modules/.pnpm/registry.npmmirror.com+@dqbd+tiktoken@1.0.7/node_modules/@dqbd/tiktoken/tiktoken_bg.wasm

    node_modules/.pnpm/registry.npmmirror.com+@dqbd+tiktoken@1.0.7/node_modules/@dqbd/tiktoken/tiktoken.js:1:22:
      1 │ import * as wasm from "./tiktoken_bg.wasm";
        ╵                       ~~~~~~~~~~~~~~~~~~~~

error during build:
Error: Build failed with 1 error:
node_modules/.pnpm/registry.npmmirror.com+@dqbd+tiktoken@1.0.7/node_modules/@dqbd/tiktoken/tiktoken.js:1:22: ERROR: No loader is configured for ".wasm" files: node_modules/.pnpm/registry.npmmirror.com+@dqbd+tiktoken@1.0.7/node_modules/@dqbd/tiktoken/tiktoken_bg.wasm
    at failureErrorWithLog (/Users/me/vscode-workspace/demo/node_modules/.pnpm/registry.npmmirror.com+esbuild@0.17.18/node_modules/esbuild/lib/main.js:1636:15)
    at /Users/me/vscode-workspace/demo/node_modules/.pnpm/registry.npmmirror.com+esbuild@0.17.18/node_modules/esbuild/lib/main.js:1048:25
    at /Users/me/vscode-workspace/demo/node_modules/.pnpm/registry.npmmirror.com+esbuild@0.17.18/node_modules/esbuild/lib/main.js:993:52
    at buildResponseToResult (/Users/me/vscode-workspace/demo/node_modules/.pnpm/registry.npmmirror.com+esbuild@0.17.18/node_modules/esbuild/lib/main.js:1046:7)
    at /Users/me/vscode-workspace/demo/node_modules/.pnpm/registry.npmmirror.com+esbuild@0.17.18/node_modules/esbuild/lib/main.js:1075:16
    at responseCallbacks.<computed> (/Users/me/vscode-workspace/demo/node_modules/.pnpm/registry.npmmirror.com+esbuild@0.17.18/node_modules/esbuild/lib/main.js:697:9)
    at handleIncomingPacket (/Users/me/vscode-workspace/demo/node_modules/.pnpm/registry.npmmirror.com+esbuild@0.17.18/node_modules/esbuild/lib/main.js:752:9)
    at Socket.readFromStdout (/Users/me/vscode-workspace/demo/node_modules/.pnpm/registry.npmmirror.com+esbuild@0.17.18/node_modules/esbuild/lib/main.js:673:7)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:324:12)
 ELIFECYCLE  Command failed with exit code 1.
```


### System Info

```shell
System:
    OS: macOS 13.3.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 161.64 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 19.1.0 - /opt/homebrew/bin/node
    npm: 8.19.3 - /opt/homebrew/bin/npm
  Browsers:
    Chrome: 113.0.5672.63
    Safari: 16.4
  npmPackages:
    @sveltejs/adapter-auto: ^2.0.0 => 2.0.1 
    @sveltejs/adapter-cloudflare: ^2.2.2 => 2.2.2 
    @sveltejs/adapter-vercel: ^2.4.3 => 2.4.3 
    @sveltejs/kit: ^1.5.0 => 1.16.2 
    svelte: ^3.54.0 => 3.59.0 
    vite: ^4.3.0 => 4.3.5
```

--- a/.changeset/honest-news-hunt.md
+++ b/.changeset/honest-news-hunt.md
@@ -1,0 +1,5 @@
+---
+'@sveltejs/adapter-cloudflare': patch
+---
+
+fix: Copy .wasm files during build

--- a/packages/adapter-cloudflare/index.js
+++ b/packages/adapter-cloudflare/index.js
@@ -54,7 +54,10 @@ export default function (options = {}) {
 				outfile: `${dest}/_worker.js`,
 				allowOverwrite: true,
 				format: 'esm',
-				bundle: true
+				bundle: true,
+				loader: {
+					'.wasm': 'copy'
+				}
 			});
 		}
 	};

